integration/src/test/java/org/kie/hacep/RemoteKieSessionImplTest.java

```diff
@@ -64,15 +67,19 @@ public void getFactCountTest() {
         kafkaServerTest.insertBatchStockTicketEvent(7,
                                                     topicsConfig,
                                                     RemoteKieSession.class);
-        try (RemoteKieSessionImpl client = new RemoteKieSessionImpl(Config.getProducerConfig("getFactCountTest"),
-                                                                    topicsConfig)) {
+        RemoteKieSessionImpl client = new RemoteKieSessionImpl(Config.getProducerConfig("getFactCountTest"),
+                                                               topicsConfig);
+        try {
             client.fireUntilHalt();
             client.listen();
             CompletableFuture<Long> factCountFuture = client.getFactCount();
-            Long factCount = factCountFuture.get(20, TimeUnit.SECONDS);
+            Long factCount = factCountFuture.get(20,
+                                                 TimeUnit.SECONDS);
             assertTrue(factCount == 7);
```

Comment:
Please use `assertEquals(7, factCount)` so in case of failure we can see in the log already the expected value

Reply:
ok